### DISCUSSION TITLE
Windows下的ClearType(文本调谐器)会影响MPV的字幕显示效果吗

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#72] [Q&A 问答]
我想给sub-font 指定思源黑体,但是据说思源黑体没有hint,在windows的这个cleartype下还不如有hint的微软雅黑...

**Comment by natural-harmonia-gropius** [comment]
默认DirectWrite应该会影响
https://docs.microsoft.com/en-us/windows/win32/directwrite/introducing-directwrite#improved-text-rendering-with-cleartype
hint无所谓，字幕的那么大的字号有hint影响也微乎其微

微软雅黑的字形其实很另类，不碰生僻字的话替换成Noble Scarlet体验更好

**Reply by yeezylife** [reply]
我装了个更纱黑体 SC(sarasa-gothic-sc-bold)
但是键入 sub-font='sarasa-gothic-sc-bold' 字体没变化... 
我发现一定得输入属性里的字体名称 支持中文.
这个字体带hint 效果貌似还行

**Reply by hingbong** [reply]
因为你用错名称了，我这么用是可以的`Sarasa UI CL Light`
`sub-font = 'Sarasa UI CL Light'`

**Reply by yeezylife** [reply]
我装的TTF,得用 sub-font='更纱黑体 SC bold'才能生效...
另外这个字体应该用哪个变种啊 我看有 UI/FIXED/TERM/GOTHIC等等.头都晕了

**Reply by hingbong** [reply]
字体INFO里面`en-us`的字体数据好像还没见过没有的，虽然不一定就是英文，兼容性应该会比较好，至于变种还是看喜好，字体repo下面都有说明了

**Reply by yeezylife** [reply]
我刚才自己也读到了

Quotes (“”) are full width —— Gothic
Quotes (“”) are narrow —— UI

UI变种引号比较窄
